torch/csrc/autograd/engine.h

```diff
@@ -110,6 +116,7 @@ struct GraphTask {
       bool keep_graph,
       bool grad_mode,
       int reentrant_depth,
+      const std::shared_ptr<ReadyQueue>& cpu_ready_queue,
```

Comment:
make this take a `std::shared_ptr<ReadyQueue>` and then move it into the field.  This combined with my earlier nit about moving into the constructor means that you only need to do one refcount bump, rather than two.